distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -799,18 +799,11 @@ public void testCswQueryWithValidationCheckerPlugin() throws Exception {
                 .all().assertThat().body(assertion[0], assertion);
         // @formatter:on
 
-        // Start metacard validation plugin; this will add on [validation-warnings = null] AND [validation-errors = null]
-        // filter to query
-        getServiceManager().startFeature(true, "catalog-plugin-metacard-validation");
```

Comment:
Yes, it was recently turned on by default in PR #979. There is no problem leaving in the call to start the feature as it would effectively be a noop. The issue was attempting to stop it.